### DISCUSSION TITLE
v3.2.55

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#11941] [General]
## What's Changed
* fix: Morph Many associate by @danharrin in https://github.com/filamentphp/filament/pull/11937
* fix: Tenant domain not matching by @danharrin in https://github.com/filamentphp/filament/pull/11940
* Improve tenancy example in docs by @zepfietje in https://github.com/filamentphp/filament/pull/11938

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.54...v3.2.55

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.55'>v3.2.55</a>.</em>

**Comment by atmonshi** [comment]
when accessing a company panel, it should show `RegisterTenant` on the url `company/new` page, 
but after this update getting 404
its look like its searching database for a company with a slug `new`
